test/runtests.jl

```diff
@@ -66,13 +68,13 @@ let
     prob = DAEProblem(resrob,u0,du0,(0.0,100000.0))
     dt = 1000
     saveat = float(collect(0:dt:100000))
-    sol = solve(prob, daskr())
-    sol = solve(prob, daskr(),save_timeseries=false)
-    sol = solve(prob, daskr(), saveat = saveat, isdiff = [true, true, false])
-    sol = solve(prob, daskr(), saveat = saveat,
+    sol = solve(prob, DASKR.Algorithm())
```

Comment:
Alright. It's a little odd for it to be the only algorithm not represented directly by the algorithm name, but it's fine.